lib/uffizzi/session_helper.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Uffizzi
+  module SessionHelper
+    class << self
+      def logged_in?
+        logged_in = ConfigFile.exists? &&
+          ConfigFile.option_exists?(:account_id) &&
+          ConfigFile.option_exists?(:cookie) &&
+          ConfigFile.option_exists?(:hostname)
+
+        puts 'You are not logged in.' unless logged_in
+
+        logged_in
```

Comment:
This is duplication of [auth_helper.rb](https://github.com/UffizziCloud/uffizzi_cli/pull/12/files#diff-4ba62c5f2ceec5068f01b0d9b718042b6194a29cb10048da667b1da62fa14819)